src/components/BoxForm/BoxForm.jsx

```diff
@@ -0,0 +1,234 @@
+import React, { useState, useEffect } from 'react';
+import {
+  FormErrorMessage,
+  Textarea,
+  FormControl,
+  FormLabel,
+  Input,
+  Checkbox,
+  Button,
+} from '@chakra-ui/react';
+import { InfoIcon } from '@chakra-ui/icons';
+import axios from 'axios';
+import isValidZipcode from 'is-valid-zipcode';
+import FYABackend from '../../common/utils';
+import DropZone from './DropZone/DropZone';
+import './BoxForm.css';
+
+function Box() {
+  const [submit, setSubmit] = useState(false);
+  const [files, setFiles] = useState([]);
+  const [formData, setFormData] = useState({
+    boxNumber: '',
+    date: '',
+    zipCode: '',
+    boxLocation: '',
+    message: '',
+    picture: '',
+    comments: '',
+  });
+
+  // states for tracking error inputs
+  const [zipCodeError, setZipCodeError] = useState(false);
+  const [dateError, setDateError] = useState(false);
+  const [boxNumberError, setBoxNumberError] = useState(false);
+
+  // useEffect ensures that the most recently updated formData is reflected
+  useEffect(async () => {
+    if (submit) {
+      // send formdata to server
+      const res = await FYABackend.post('/boxForm', formData, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+      console.log(res.data);
+      setSubmit(false);
+    }
+  }, [submit]);
+
+  const { boxNumber, date, zipCode, boxLocation, message, comments } = formData;
+
+  const uploadBoxPhoto = async file => {
+    // get S3 upload url from server
+    const { data: uploadUrl } = await FYABackend.get('/s3Upload');
+
+    // upload image directly to S3 bucket
+    await axios.put(uploadUrl, file, {
+      headers: {
+        'Content-Type': 'multipart/form-data',
+      },
+    });
+
+    // return box image url
+    const imageUrl = uploadUrl.split('?')[0];
+    return imageUrl;
+  };
+
+  const isValidZip = zip => {
```

Comment:
This is awesome! Could we move this to the utils file? I'm sure other box forms will need this function as well.